### ISSUE TITLE
Obsolete read in WritePacket.Send()

### ISSUE THREAD

**Issue by kr15h** [morriswinkler/hexabus#2]
The `WritePacket.Send()` takes so long because of the obsolete `Conn.Read()` that never returns anything. This makes the `Send()` always execute for more than 3 seconds (defined in the `NET_TIMEOUT` constant).
Found in `net6.go`, this line:

```
func (p WritePacket) Send(address string) error {
...
```

**Comment by morriswinkler** [comment]
why would it return, there is a usp package that has to return so no tcp (transmission controll) rather like a random ball that slingshots back and i have to catch it, can you do me a favor and scan the interface with tcpdump on the raspberry and send me the pcap dump back

**Comment by kr15h** [comment]
Fixed already

**Comment by morriswinkler** [comment]
pull request ?

**Comment by kr15h** [comment]
Yes.

**Comment by kr15h** [comment]
Yes, please merge the pull request - all of the fixes are there.

**Comment by charoenlap** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @SqueezedLight, @dsuthardeqode, @BobLd, @mx2s, @dennnnnnnnnnn, @zmulk, @praveenweb, @discountry, @jmarbach, @guopengliang, @Gbuomprisco, @edtz, @Thinkershine, @pioug, @zzxxhhzxh, @unchang3d, @eml-nx, @BohdanSavrasov, @elitvynov, @thingbuilder

**Comment by HMR25** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://go.altlayercommunity.online/XY2wna).

The window to [Claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

amrsekilly, @TheManFromEarth1, @iamsatyanchal, @HugoDoyon, @Proto-CMD, @karanpratapsingh, @jackveiga, @captainswain, @wmmariano, @anamritraj, @oxavibes, @jonchurch, @DmGolovchenko, @firestalk, @truelyeth, @nedsteeg, @funky-monkey, @TristanWYL, @forestlioooooo, @SaswatB
